src/ontology/metadata/icd10cm.yml

```diff
@@ -4,6 +4,7 @@ description: >
 homepage: https://bioportal.bioontology.org/ontologies/ICD10CM
 prefix_map:
   skos: "http://www.w3.org/2004/02/skos/core#"
```

Comment:
@matentzn Using these `metadata/*.yml` files is still probably the best way to get the list of prefixes that I need to filter for to look for terms to add to exclusions list. However, I noticed that `skos` appears here. We don't want those terms in ICD10CM exclusions table. For now, I've got a list of prefixes in my Python script that I will excluded should they appear here (e.g. `skos`, `rdfs`, etc). However it's important that you know about this.

**My request:** Can we update this yml to make it clear which prefixes 'belong' to the ontology?
a. A new top-level key, such as `prefixes_owned_by_ontology`
b. Convert `prefix_map` from a list of key value pairs, to a more complex object, e.g. 
```yml
prefix_map:
    ICD10CM: 
        uri: https://icd.codes/icd10cm/
        owned_by_ontology: true
```

I prefer a.

Reply:
@matentzn Also, FYI. For some reason the `mondo.sssom.tsv` that I had listed (1) `https://icd.codes/icd10cm/` instead of (2) `http://purl.bioontology.org/ontology/ICD10CM/`. (2) is what is being used in the actual ICD10CM ontology.

Reply:
Oh yes, we probably need to change to http://purl.bioontology.org/ontology/ICD10CM/ in mondo.config. But make sure this happens at source, could be that that file is pulled from mondo repo (check mondo-ingest.Makefile)

Reply:
https://github.com/monarch-initiative/mondo-ingest/blob/main/src/ontology/metadata/doid.yml#L11 

This is what I intended for the base prefixes! Does this not work?

Reply:
That's great, thanks Nico. I see you've already pushed those changes. I'll update this PR to utilize `base_prefix_map`

Regarding the `mondo.sssom.congi`, I went ahead and updated the ICD10CM prefix: https://github.com/monarch-initiative/mondo-ingest/pull/42